shared/actions/tracker.js

```diff
@@ -452,6 +452,25 @@ function _serverCallMap(
     }
   }
 
+  // We queue up responses and handle them one at a time
+  let _idleResponseQueue = []
+
+  const addToIdleResponseQueue = (f: () => void) => {
+    _idleResponseQueue.push(f)
+    requestIdle(onRequestIdleQueueHandler)
+  }
+
+  const onRequestIdleQueueHandler = () => {
+    if (!_idleResponseQueue.length) {
+      return
+    }
+    const toHandle = _idleResponseQueue.pop()
```

Comment:
k i did this (but i use a do while so we at least do a pop once), also i updated our fallbacks to give android a little more breathing room (20ms vs 1). in dev mode on ios simulator the value is always 0 time remaining..., anyways it'll effectively fallback to what i had before anyways now, ptal